lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -41,13 +41,26 @@ def setUp(self):
     def test_pass_proctoring_tab_in_instructor_dashboard(self):
         """
         Test Pass Proctoring Tab is in the Instructor Dashboard
+        for global staff and course staff(role of STAFF or ADMIN)
         """
         self.instructor.is_staff = True
         self.instructor.save()
 
-        response = self.client.get(self.url)
-        self.assertIn(self.proctoring_link, response.content)
-        self.assertIn('Allowance Section', response.content)
+        self.has_proctoring_tab()
+
+        CourseFinanceAdminRole(self.course.id).remove_users(self.instructor)
```

Comment:
how `CourseFinanceAdminRole` effects the tab from rendering?

Reply:
It has nothing to do with the tab.